libr/cons/cons.c

```diff
@@ -696,12 +715,25 @@ R_API void r_cons_flush() {
 		r_cons_reset ();
 		return;
 	}
+ 	if (lastMatters ()) {
+		if (!CTX (lastMode)) {
+			// snapshot of the output
+			free (CTX (lastOutput));
+			CTX (lastOutput) = calloc (CTX (buffer_len) + 1, 1);
+			if (CTX (lastOutput)) {
+				CTX (lastLength) = CTX (buffer_len);
```

Comment:
I don't think you need this, since `lastLength` is always equal to `buffer_len` you can just use the latter.

Reply:
i need it for the r_cons_last() which prints the last buffer to be flushed laster, in that case lastMode is set so its not captured again and the same cache is used.